tasks/main.yml

```
@@ -0,0 +1,41 @@
+---
+- name: Gather OS specific variables
+  include_vars: "{{ loop_vars }}"
+  loop: "{{ query('first_found', params) }}"
+  vars:
+    params:
+      files:
+        - "{{ ansible_distro }}-{{ ansible_distro_version }}.yml"
+        - "{{ ansible_distro }}-{{ ansible_distro_release }}.yml"
+        - "{{ ansible_distro }}-{{ ansible_distro_major_version }}.yml"
+        - "{{ ansible_distro }}.yml"
+        - "{{ ansible_os_family|lower }}-family.yml"
+        - "{{ ansible_system | lower }}.yml"
+      paths:
+        - "vars"
+      skip: true
+  loop_control:
+    loop_var: loop_vars
+
+- name: reload services before certbot tasks
+  include_tasks: reload-services.yml
+  when:
+    - certbot_reload_services | length > 0
+    - certbot_reload_services_before_enabled
+
+- include_tasks: install-certbot.yml
+  tags: certbot_install
+
+- include_tasks: create-cert-webroot.yml
+  with_items: "{{ certbot_certs }}"
+  loop_control:
+    loop_var: cert_item
```

```suggestion
    loop_var: _certbot_cert_item
```

Private variables should be prefixed with `_certbot_`